test/z_cli_daemon_test.go

```diff
@@ -470,6 +470,47 @@ func (suite *PouchDaemonSuite) TestDaemonWithMultiRuntimes(c *check.C) {
 	dcfg2.KillDaemon()
 }
 
+// TestRestartStoppedContainerAfterDaemonRestart is used to test the case that
+// when container is stopped and then pouchd restarts, the restore logic should
+// initialize the existing container IO settings even though they are not alive.
+func (suite *PouchDaemonSuite) TestRestartStoppedContainerAfterDaemonRestart(c *check.C) {
+	cfg := daemon.NewConfig()
```

Comment:
Is start daemon with config file? or with command line as before?
And we need think about alios test.

Reply:
synced with @rudyfly offline. @rudyfly is right. we would use specific config file when we write daemon related case. If not, the `/etc/pouch/config.json` will impact the running case. 

updated it.